src/mapblock.h

```diff
@@ -498,13 +498,11 @@ class MapBlock
 
 	/*
 		Used only internally, because changes can't be tracked
+		Before use check that isValidPosition(x, y. z) returns
+		"true".
 	*/
-
 	inline MapNode &getNodeRef(s16 x, s16 y, s16 z)
```

Comment:
Now there are no bound checks left over, and the function name should indicate that.
For example: `getNodeUnsafe`.

Reply:
Will use "getNodeRefNoCheck" as this is the style for "unsafe" methods throughout the code.